lib/zosAccessor.js

```diff
@@ -692,7 +787,7 @@ ZosAccessor.prototype.getJobLog = function (jobName, jobId, logFormat) {
                 });
                 readStream.on('end', function () {
                     var logString = Buffer.concat(chunks).toString('utf8');
-                    if (logFormat === 'metaInfo') {  // Deprecated
+                    if (fileId === 'metaInfo') {  // Deprecated
```

Comment:
Altough it is deprecated, It should be`option.fileId` instead of `fileId`, right?

Reply:
The `option.fileId` is changed to `'x'`. Here we check the original fileId passed from parameter. We'll drop `'metaInfo'` in 2.0.0.